src/RenderWebGL.js

```diff
@@ -182,9 +182,23 @@ class RenderWebGL extends EventEmitter {
         /** @type {function} */
         this._exitRegion = null;
 
+        /** @type {object} */
+        this._backgroundDrawRegionId = {
+            enter: () => this._enterDrawBackground(),
+            exit: () => this._exitDrawBackground()
+        };
+
         /** @type {Array.<snapshotCallback>} */
         this._snapshotCallbacks = [];
 
+        /** @type {Array<number>}
+         * @readonly */
+        this._backgroundColor4f = [0, 0, 0, 0];
+
+        /** @type {Uint8ClampedArray}
+         * @readonly */
+        this._backgroundColor3b = new Uint8ClampedArray(3);
+
```

Comment:
I didn't remove `this._backgroundColor` because it never existed here-- it's first defined when `setBackgroundColor` is called